lib/spotlight_search/exportable_columns_v2.rb

```diff
@@ -32,6 +32,10 @@ def export_columns(*record_fields, **associated_fields)
         self.enabled_columns = [*record_fields, **associated_fields]
       end
 
+      def default_scopes_for_export(*filter_scopes)
```

Comment:
If an array is needed, `filter_scopes` is enough bro. [*filter_scopes] and filter_scopes are same if it's an array.